wg-api/spec-documents/scroll-view.md

```diff
@@ -0,0 +1,256 @@
+# Scroll View
+
+## Summary
+
+This spec proposes to add an experimental native ScrollView component to Electron.
```

Comment:
docs: all these APIs should be flagged with `_Experimental_`

Reply:
fixed